app/javascript/components/components/Header.js

```diff
@@ -10,29 +29,56 @@ class Header extends Component {
       sign_in_route,
       sign_out_route
     } = this.props
-    console.log("logged_in:", logged_in)
-    console.log("current_user:", current_user)
     return (
-      <>
-        <h1>This is a Header</h1>
-        <Nav>
-          {logged_in &&
-            <NavItem>
-              <a href={sign_out_route} className="nav-link">Sign Out</a>
-            </NavItem>
-          }
-          {!logged_in &&
-            <NavItem>
-              <a href={sign_in_route} className="nav-link">Sign In</a>
-            </NavItem>
-          }
-          {!logged_in &&
-            <NavItem>
-              <a href={new_user_route} className="nav-link">Sign Up</a>
-            </NavItem>
-          }
-        </Nav>
-      </>
+      <header>
+        <Navbar
+          color="secondary"
+          expand="md"
+          light
+        >
+          <NavbarBrand href="/">
+          Apartments Department
+          </NavbarBrand>
+          <NavbarToggler onClick={this.toggle} />
+          <Collapse isOpen={this.state.isOpen} navbar>
+            <Nav
+            className="me-auto"
+            navbar
+            >
+              <NavItem>
+                <NavLink href="/">
```

Comment:
Same thing here with your NavLink to home and to apartment index. Your devise routes are fine and do not need to be changed.